public/js/movieDetails.js

```diff
@@ -0,0 +1,361 @@
+import { createNav, genres, createElement, fetch_function } from '../utils/pageCreation.js';
+
+async function detailsSection(movie) {
```

Comment:
Use verbs / action words when naming functions. For example `createDetailsSection` or `renderDetailsSection`